Luke7787/ImmaculateInventors
# Feature/remove_items

Summary:
-can remove items if quantity is less than or equal to 0

How to Test:
-add user to database (POST)http://localhost:8000/users
{
    "username": "testName",
    "password": "Password123!"
}
-get that user's id (GET)http://localhost:8000/users
-add item to the user (PATCH)http://localhost:8000/itemToUser?uid=[userID]
{
    "userId": "userID",
    "name": "itemName",
    "quantity": 3
}
-try decrementing or incrementing the quantity
(PATCH)http://localhost:8000/items?uid=[userID]&id=[itemID]&option=[sub/add]&quantity=3
try to decrement with a number that makes the quantity = 0 and check if the item id is still in the user list
